picard/tagger.py

```diff
@@ -453,29 +453,26 @@ def _file_loaded(self, file, target=None, remove_file=False):
                 self.move_file_to_nat(file, recordingid)
                 return
 
-        if target:
-            self.move_file(file, target)
-        else:
-            self.unclustered_files.add_file(file)
+        self.move_file(file, target)
 
         # fallback on analyze if nothing else worked
         if config.setting['analyze_new_files'] and file.can_analyze():
             log.debug("Trying to analyze %r ...", file)
             self.analyze([file])
 
     def move_file(self, file, target):
-        if target is None:
-            log.debug("Aborting move since target is invalid")
```

Comment:
What if target is None and file.parent? Shouldn't we (debug) log this case or is it handled somewhere?

Reply:
In current code it actually should never happen that file has a parent as the `move_file` method is only used for new files. But yes, I intentionally made this function so it could be used more generally without breaking. Target None and file having a parent would mean the file stays where it is. Could make sense to log this for debugging, I'll add it.